media-gfx/gpicview/gpicview-0.2.5-r1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils xdg-utils
+
+DESCRIPTION="A Simple and Fast Image Viewer for X"
+HOMEPAGE="http://lxde.sourceforge.net/gpicview"
+SRC_URI="mirror://sourceforge/lxde/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~x86 ~arm-linux ~x86-linux"
+IUSE=""
+
+RDEPEND="virtual/jpeg:0
+	>=x11-libs/gtk+-2.6:2"
+DEPEND="${RDEPEND}
+	>=dev-util/intltool-0.40
+	virtual/pkgconfig
+	sys-devel/gettext"
+
+src_install() {
+	emake DESTDIR="${D}" install
+	dodoc AUTHORS
+}
```

Comment:
src_install is unnecessary. It doesn't deviate from the default. Go ahead and delete it.